tests/outbound-redis/http-rust-outbound-redis/src/lib.rs

```diff
@@ -28,10 +30,99 @@ fn test(_req: Request) -> Result<Response> {
 
     let keys = vec!["spin-example-get-set", "spin-example-incr"];
 
-    let del_keys = redis::del(&address, &keys)
-        .map_err(|_| anyhow!("Error executing Redis incr command"))?;
+    let del_keys =
+        redis::del(&address, &keys).map_err(|_| anyhow!("Error executing Redis incr command"))?;
 
     assert_eq!(del_keys, 2);
 
+    redis::execute(
+        &address,
+        "set",
+        &[
+            ValueParam::String("spin-example"),
+            ValueParam::Data(b"Eureka!"),
+        ],
+    )
+    .map_err(|_| anyhow!("Error executing Redis set command"))?;
+
+    redis::execute(
+        &address,
+        "append",
+        &[
+            ValueParam::String("spin-example"),
+            ValueParam::Data(b" I've got it!"),
+        ],
+    )
+    .map_err(|_| anyhow!("Error executing Redis set command"))?;
+
+    let values = redis::execute(&address, "get", &[ValueParam::String("spin-example")])
+        .map_err(|_| anyhow!("Error executing Redis get command"))?;
```

Comment:
As there is already a named `get` command, would be be useful in this message to specify that the error was in executing it _via_ the `execute` command?